c7n/resources/backup.py

```diff
@@ -55,3 +57,38 @@ def get_resources(self, resource_ids, cache=True):
             except client.exceptions.ResourceNotFoundException:
                 continue
         return resources
+
+
+@resources.register('backup-vault')
+class BackupVault(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'backup'
+        enum_spec = ('list_backup_vaults', 'BackupVaultList', None)
+        detail_spec = ('describe_backup_vault', 'BackupVaultName', 'BackupVaultName', None)
```

Comment:
afaics we probably don't need the detail spec, the enum spec which lists has the same information, we do detail specs when enum just returns limited information